spec/integration/orders_spec.rb

```diff
@@ -37,6 +37,7 @@
     order = create_order_response.data
     expect(create_order_response.success).to eq true
     expect(order.id).not_to be_nil
+    expect(order.created_at).to be_a_kind_of(Time)
```

Comment:
With the correct type, the SDK does all the parsing and automatically returns a Time object. Success!